src/environment.js

```diff
@@ -122,9 +123,11 @@ if (process.env.NODE_ENV !== 'production') {
 
 export const defaultEthNode =
   getDefaultEthNode() || networkConfig.nodes.defaultEth
+const defaultEthProvider = new Web3.providers.WebsocketProvider(defaultEthNode)
```

Comment:
Shall we use it for `defaultEthProvider` as well, since it can reconnect?

Reply:
No idea, shall we @sohkai?